### PR TITLE
Attempt to make a DaSL/ITCR combo template

### LINKED ISSUE #80: Thoughts: Keeping this and DaSL template updated concurrently is not great
With only a few days of having DaSL and this template going, I'm finding the maintenance of both is a bit redundant and I'm worried about something falling through the cracks. 

Basically whenever I make a change here I have to make a change over on the DaSL and its not a great system. 
Not sure how or if there's a way to make this better, but I'm going to write this down in case anyone has ideas.

### PR DESCRIPTION
**Pull request by cansavvy**
If we like this solution, it will close #80. Basically keeping updates going on both the DaSL and ITCR versions of these repos is not a great long term solution and requires redundant work. So I'm attempting to see if we can go back to one template bookdown repo. 

I've followed what I described here: https://github.com/jhudsl/ITCR_Course_Template_Bookdown/issues/80#issuecomment-830069599 so we can see how this looks. 

Basically the getting_started.md has an extra section about the google slides and also about deleting alternative files.

I added these steps to the issues that get filed too. 

How do we feel about this set up?

### CODE REVIEW

**Comment by cansavvy** [comment]
@KatherineCox, do you want to weigh in on this? If this solution is okay with everyone, I would end up deleting the  current https://github.com/jhudsl/DaSL_Course_Template_Bookdown repo and making this a more general template repo for all purposes.

**Review by carriewright11 — APPROVED** [review]
Looks good! I will  make a pull request with some updates to the logo section after this is merged.
